src/test/run-make/native-link-modifier-bundle-split/Makefile

```diff
@@ -0,0 +1,35 @@
+# ignore-cross-compile
+# ignore-windows-msvc
+
+-include ../../run-make-fulldeps/tools.mk
+
+# We're using the llvm-nm instead of the system nm to ensure it is compatible
+# with the LLVM bitcode generated by rustc.
+NM = "$(LLVM_BIN_DIR)"/llvm-nm
+SPLIT = "-Zsplit-bundled-libs"
+BUNDLED_LIB = "libbundled.rlib.bundle.native-staticlib.a"
+
+all: $(call NATIVE_STATICLIB,native-staticlib)
+	# Build a staticlib and a rlib, the `native_func` symbol will be bundled only into staticlib
+	$(RUSTC) bundled.rs --crate-type=staticlib --crate-type=rlib $(SPLIT)
+	$(NM) $(TMPDIR)/libbundled.a | $(CGREP) -e "T _*native_func"
+	$(NM) $(TMPDIR)/libbundled.a | $(CGREP) -e "U _*native_func"
+	$(NM) $(TMPDIR)/libbundled.rlib | $(CGREP) -ve "T _*native_func"
+	$(NM) $(TMPDIR)/libbundled.rlib | $(CGREP) -e "U _*native_func"
+
+	# Build a staticlib and a rlib, the `native_func` symbol will not be bundled into it
+	$(RUSTC) non-bundled.rs --crate-type=staticlib --crate-type=rlib $(SPLIT)
+	$(NM) $(TMPDIR)/libnon_bundled.a | $(CGREP) -ve "T _*native_func"
+	$(NM) $(TMPDIR)/libnon_bundled.a | $(CGREP) -e "U _*native_func"
+	$(NM) $(TMPDIR)/libnon_bundled.rlib | $(CGREP) -ve "T _*native_func"
+	$(NM) $(TMPDIR)/libnon_bundled.rlib | $(CGREP) -e "U _*native_func"
```

Comment:
Looks like the non-bundled case is already tested in a different test in exactly the same form, so we probably don't need to duplicate it here.

Reply:
(Maybe the split-bundled-libs case could also be just added to `src\test\run-make\native-link-modifier-bundle\Makefile` instead of creating a separate test, if all the .c and .rs files are identical.)